app/components/button/button.tsx

```diff
@@ -19,6 +19,7 @@ const variantClasses = {
   outline: "border border-sky-200 text-sky-500 hover:bg-sky-100 shadow",
   danger: "bg-red-500 hover:bg-red-700 text-white",
   cancel: "bg-white ring-1 ring-black/10 hover:bg-black/5 text-gray-900",
+  review: "border border-gray-200 text-stone-00",
```

Comment:
a more generic name would be better here other than "review", `gray` `dull` ... idk